help/en/package/jmri/jmrit/operations/Operations.shtml

```diff
@@ -603,19 +603,16 @@
                   <a href="#CopyTrain">Copy Train</a>
                 </li>
 
-                <li>
-                  <a href="#TrainShowCarTypesServiced">Show Car Types Serviced (Diagnostic
-                  Tool)</a>
-                </li>
+        <li><a href="#TrainShowCarTypesServiced">Show Car Types Serviced (Diagnostic Tool)</a>
```

Comment:
No need to change, but the indentation seem to have gone wrong with these changes.

Reply:
Yes, I saw that.  Didn't want to reformat the entire section, so I let it go.